### DISCUSSION TITLE
How to run/attach PrettyZoo to zookeeper?

### DISCUSSION THREAD

**Discussion by zxcvwrs** [vran-dev/PrettyZoo#210] [Q&A] [answered]
Hello, I'm a newbie to Linux systems and zookeeper. I've installed PrettyZoo by doing `$sudo dpkg -i prettyzoo_1.9.3-1_amd64.deb`, yet I really don't know how to attach PrettyZoo to ZooKeeper or run PrettyZoo. Can someone help me with walkthrough on how to run PrettyZoo on Ubuntu?

**Comment by tigersizer** [comment, marked as answer]
Same question from CentOS 8.
```
$ sudo rpm -i prettyzoo-1.9.4-1.x86_64.rpm 
$ prettyzoo
bash: prettyzoo: command not found...
```
You can figure it out from (obviously will not work on a .deb file with dpkg, but I'm sure the Intertubes will tell you how to find out what is inside):
```
$ sudo rpm -ql prettyzoo
```
For whatever reason, it installed itself in <tt>/opt/prettyzoo/bin/prettyZoo</tt>.
Gtk is not entirely happy (<tt>Failed to load module "canberra-gtk-module"</tt>), but it fired up just fine.

You can make life easier with:
```
$ echo "/opt/prettyzoo/bin/prettyZoo" > ~/bin/prettyZoo
$ chmod +x ~/bin/prettyZoo
```
You may need to "mkdir ~/bin", if you haven't used it before.
You may need to add ~/bin to the PATH in ~/.bashrc (CentOS does that magically; don't know about other distros)
